src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+
+        System.out.println(logo);
+
+        String line = "    ____________________________________________________________\n";
+
+        Task[] tasks = new Task[100];
+        Scanner in = new Scanner(System.in);
+        String action = in.nextLine();
+        int index = 0;
+        String bye = "bye";
+
+        while (!action.equals(bye)) {
+            System.out.println(line);
+
+            if (action.equals("list")) {
+                System.out.println("     Here are the tasks in your list:\n");
+                for (int i = 0; i < index; i = i + 1) {
+                    int num = i + 1;
+                    System.out.println("     " + num
+                                        + ".[" + tasks[i].getTypeOfTask()
+                                        + "][" + tasks[i].getStatusIcon()
+                                        + "] " + tasks[i].getDescription());
+                }
+            } else if (action.startsWith("mark")) {
+                String[] inputs = action.split(" ");
+                int ind = Integer.parseInt(inputs[1]) - 1;
+                tasks[ind].mark();
+                System.out.println("     Nice! I've marked this task as done:\n");
+                System.out.println("       "
+                                    + "[" + tasks[ind].getStatusIcon() + "] "
+                                    + tasks[ind].getDescription());
+
+            } else if (action.startsWith("unmark")) {
+                String[] inputs = action.split(" ");
+                int ind = Integer.parseInt(inputs[1]) - 1;
+                tasks[ind].unmark();
+                System.out.println("     OK, I've marked this task as not done yet:\n");
+                System.out.println("       "
```

Comment:
Perhaps it might be better to split the user input before the if-else statements to prevent repetitive code such as 
String[] inputs = action.split(" ");
int ind = Integer.parseInt(inputs[1]) - 1; 
Other than that, conditions are well set.